client/src/components/MultiJurisdictionAudit/useJurisdictions.ts

```diff
@@ -22,6 +22,7 @@ export enum JurisdictionRoundStatus {
   NOT_STARTED = 'NOT_STARTED',
   IN_PROGRESS = 'IN_PROGRESS',
   COMPLETE = 'COMPLETE',
+  FAILED = 'FAILED',
```

Comment:
`FAILED` isn't a status that gets returned from the server, so i think it's a bit confusing to have it be part of this type. this type represents the jurisdiction's status with regard to the ballots it's been assigned to audit during the round.

two ideas for what `getJurisdictionStatus` should return:
- a union of `JurisdictionRoundStatus | FileProcessingStatus`
- a new enum `JurisdictionProgressStatus` that has states for file uploads (e.g. `UPLOADS_NOT_STARTED`, `UPLOADS_IN_PROGRESS`) as well as for auditing progress (`AUDITING_NOT_STARTED`, `AUDITING_IN_PROGRESS`)